modules/data-entry/src/main/java/io/uhndata/cards/DataImportServlet.java

```diff
@@ -782,6 +784,7 @@ private Node findSubject(String subjectKey, String subjectId, Node typeNode, Nod
         } catch (RepositoryException ex) {
             // No change to query
         }
+        query += " OPTION (index tag property)"
```

Comment:
How did this omission happen?...

```suggestion
        query += " OPTION (index tag property)";
```